src/etc/pre-push.sh

```diff
@@ -15,3 +15,7 @@ echo "Running pre-push script $ROOT_DIR/x test tidy"
 
 cd "$ROOT_DIR"
 ./x test tidy --set build.locked-deps=true
+if [ $? -ne 0 ]; then
```

Comment:
```suggestion
res=ok
./x test tidy --set build.locked-deps=true || res=fail
if [ "$res" != 0 ]; then
```

Probably too late, but you could do something like this to not need to disable the `-e` flag. Not really needed anyway in this tiny script though

Reply:
nice trick!
but it should be:
```bash
if [ "$res" != "ok" ]; then
```
right?

Reply:
Whoops, you're right!